static/js/src/advantage/react/components/Subscriptions/Notifications/Notifications.test.tsx

```diff
@@ -7,6 +7,7 @@ import {
   userSubscriptionFactory,
   userSubscriptionStatusesFactory,
 } from "advantage/tests/factories/api";
+import { OfferFactory } from "advantage/offers/tests/factories/offers";
 
 describe("Notifications", () => {
```

Comment:
This is a very generic description of the tests. Could you make it specific to the feature?